### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,39 +1,66 @@
...
 
-<h3>Description</h3>
+    ```js
+    var allowEdit = false;
+    function OnAllowEditChanged(s, e) {
+        allowEdit = s.GetValue();
+    }
+    ```
+
+3. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing), [BatchEditRowInserting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowInserting), and [BathEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events. In the handler, cancel the current editing operation based on the flag variable value.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > [BathEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events.

"Bath" instead of "Batch". Another typo that is easily missed by any spellchecker. Please be more attentive.

> cancel the current editing operation

Is there any specific reason to use the gerund form? The word "edit" works well enough as an adjective. The current edit operation.

```suggestion
3. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing), [BatchEditRowInserting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowInserting), and [BatchEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events. In the handler, cancel the current edit operation based on the flag variable value.
```

File: Readme.md [hunk 2]
```diff
@@ -1,39 +1,66 @@
...
+        <Columns>
+            <dx:GridViewCommandColumn ShowNewButtonInHeader="true" ShowDeleteButton="true" />
+            <!-- ... -->
+        </Columns>
+    </dx:ASPxGridView>
+    ```
 
-<p>This example demonstrates how to conditionally enable or disable the editing on the client side.</p>
-<p><strong>See also:</strong><br><a href="https://www.devexpress.com/Support/Center/p/T150965">GridView - Batch Editing - How to conditionally enable and disable the editing on the client side</a></p>
+2. Add a check box and handle its client-side [CheckedChanged](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxCheckBox.CheckedChanged) event. In the handler, create a flag variable, get the current state of the check box, and assign this value to the flag variable.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > In the handler, create a flag variable, get the current **state** of the check box, and assign this **value** to the flag variable.

Implied synonyms. You used "value" as a synonym of "state" but you never introduced them as such. Since you've said "get the state", it already implies that it is a value. Therefore, we can refer to this value as a "state" in the "assign" context:

```suggestion
2. Add a check box and handle its client-side [CheckedChanged](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxCheckBox.CheckedChanged) event. In the handler, create a flag variable, get the current state of the check box, and assign the state to the flag variable.
```

File: Readme.md [hunk 3]
```diff
@@ -1,39 +1,66 @@
...
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView- How to conditionally enable and disable the batch editing on the client side
+# Grid View for ASP.NET Web Forms - How to control data editing on the client side in batch edit mode
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: I'd like to question your choice of words here.

> How **to control** data editing

The word "to control" implies much, it can be actually anything. You can modify user input on-the-fly, you can prevent users from editing something, etc. However, this topic covers a very specific scenario, namely when you forbid users from editing a cell based on the current check box state. This topic does not explain how to assume the general control over user input.

The original topic name was good and bad at the same time. We could enhance it instead:

```suggestion
# Grid View for ASP.NET Web Forms - How to enable or disable the cell edit functionality in batch mode based on a condition
```

This is just a suggestion. You can come up with something better instead.

File: Readme.md [hunk 4]
```diff
@@ -1,39 +1,66 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t150957/)**
 <!-- run online end -->
 
+This example demonstrates how to use the check box state to control data editing in the grid in batch edit mode.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]:
```suggestion
This example demonstrates how to use the check box state to enable or disable the grid's cell edit functionality in batch edit mode.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
**Structure and Semantics**

This is a simple readme topic for a code example targeting a very specific grid usage scenario. The structure is clean, straightforward, and has no clarity issues.

**Images**

The included GIF animation is clean and clear. Well done.

**Language**

In general, your language is good and follows the best practices accepted for DevExpress documentation production pipelines. I found only the following minor issues:

1. A typo. Please mind the typos, not all typos are noticed by a spellchecker.
2. Implied synonyms. That is, when you speak of something first (state), and then introduce a synonym for it (value) assuming that it's already clear and well-understood. You made no earlier introduction that "state" and "value" are the same thing. Yes, one might guess what does it mean but the purpose of our documentation is to require as little mental effort as possible.
3. Generalizations. What does it mean "to control data editing"? It may mean way too many things, however, this particular topic covers a narrow subset of this "control". I believe we should be more specific upfront.

**Bottom Line**

This is a good topic that meets all quality criteria of DevExpress documentation. Don't rest easy. Keep it up.

File: Readme.md (hunk 3 above)
Comment (elenapeskova) [reply Readme.md]: I like your variant and absolutely agree with your comments. Since this scenario is a specific one, we should tell a user about it. Thank you!

Comment (dmitry-eliseev-devexpress) [reply Readme.md]: You are always welcome. Every time you use a word, you should ask yourself -- is this word clear and unambiguous? Does it serve its purpose well in the current context? If you can answer "yes" to the both questions -- it means you've chosen the right word for the task. If not -- it's probably worth taking another look.

File: Readme.md [hunk 5]
```diff
@@ -1,39 +1,66 @@
...
-    e.cancel = !allowEdit;
-}</code>
+## Documentation
 
-<br/>
+* [Grid in Batch Edit Mode](https://docs.devexpress.com/AspNet/16443/components/grid-view/concepts/edit-data/batch-edit-mode)
 
+## More Examples
 
+* [Grid View for ASP.NET MVC - How to control data editing on the client side in batch edit mode](https://github.com/DevExpress-Examples/gridview-how-to-conditionally-enable-and-disable-the-batch-editing-on-the-client-side-t150965)
```
Comment (elenapeskova) [inline Readme.md]:
```suggestion
* [Grid View for ASP.NET MVC - How to enable or disable the cell edit functionality in batch mode based on a condition](https://github.com/DevExpress-Examples/gridview-how-to-conditionally-enable-and-disable-the-batch-editing-on-the-client-side-t150965)
```
